packages/core/content-manager/server/controllers/collection-types.js

```diff
@@ -329,6 +329,50 @@ module.exports = {
 
     const number = await entityManager.getNumberOfDraftRelations(id, model);
 
+    return {
+      data: number,
+    };
+  },
+  async getMultipleEntriesNumberOfDraftRelations(ctx) {
+    const { userAbility } = ctx.state;
+    const { model, ids } = ctx.params;
+
+    const entityManager = getService('entity-manager');
+    const permissionChecker = getService('permission-checker').create({ userAbility, model });
+
+    if (permissionChecker.cannot.read()) {
+      return ctx.forbidden();
+    }
+
+    const permissionQuery = await permissionChecker.sanitizedQuery.read(ctx.query);
+    const populate = await getService('populate-builder')(model)
+      .populateFromQuery(permissionQuery)
+      .build();
+
+    const idsArray = JSON.parse(ids);
```

Comment:
Passing an array of ids should be done via query params and not a path param.

Reply:
done